src/commands/instance/spawn.ts

```diff
@@ -203,35 +204,36 @@ export default class InstanceSpawn extends Kommand {
     this.log(chalk.grey("Checking prerequisites..."));
     const checks: Listr = new Listr([
       {
-        title: `docker-compose exists and the version is at least ${MIN_DOCO_VERSION}`,
+        title: `docker compose exists and the version is at least ${MIN_DOCO_VERSION}`,
         task: async () => {
           try {
-            const docov = await execute("docker-compose", "-v");
+            const docov = await execute("docker", "compose version");
             const matches = docov.stdout.match(/[^0-9.]*([0-9.]*).*/);
+
             if (matches === null) {
               throw new Error(
-                "Unable to read docker-compose verson. This is weird."
+                "Unable to read docker compose verson. This is weird."
               );
             }
             const docoVersion = matches.length > 0 ? matches[1] : null;
 
             if (docoVersion === null) {
               throw new Error(
-                "Unable to read docker-compose version. This is weird."
+                "Unable to read docker compose version. This is weird."
```

Comment:
Same comment as above about refactoring.

```suggestion
                "Unable to read the version of Docker Compose. This is weird."
```